apps/prometheus/helm-release.yaml

```diff
@@ -15,3 +15,14 @@ spec:
   values:
     grafana:
       enabled: false
+    prometheus:
+      prometheusSpec:
+        replicas: 3
+        retention: 14d
+        storageSpec:
+          volumeClaimTemplate:
+            spec:
+              storageClassName: local-path
+              resources:
+                requests:
+                  storage: 1Gi
```

Comment:
Is this sufficient? I get about 2Gi per week in prometheus but I might have more metrics scraped

Reply:
I don't know, let's see?